overlord/snapstate/can_remove.go

```diff
@@ -17,29 +17,26 @@
  *
  */
 
-package snappy
+package snapstate
```

Comment:
well it can go in snapstate.go itself, it's small enough, util feels wrong because this is main policy